docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -34,103 +34,240 @@ Let us walk through the changes that are required in different phases of your ne
-    Let's say a Fabric network consists of two organizations called `myorg1` and `myorg2`, each running a Layer-2 application with a web server whose URL prefixes are `http://myorg1.mynetwork.com:9000` and `http://myorg2.mynetwork.com:9000` respectively. For the configuration associated with the channel `mychannel`, each app exposes a REST endpoint (again, as an example) `http://myorg1.mynetwork.com:9000/mychannel/org_sec_group` and `http://myorg2.mynetwork.com:9000/mychannel/org_sec_group` respectively.
-    At each web server's backend, you need to implement logic to retrieve the organization's MSP ID and its associated certificated chains. Sample code is given below for a JavaScript implementation built on `fabric-sdk-node`. You can use this code verbatim if your Layer-2 application is built on JavaScript or TypeScript, or port it to other languages (Java or Golang).
-    ```javascript
-    var express = require('express');
-    var app = express();
-    
-    // This is boilerplate code your Layer-2 application is already likely to implement
-    // 'userId' represents a wallet identity
-    var getNetworkForOrgUser = async function(userId) {
-        let wallet = await getWalletForOrg();
-	    try {
-		    const connectionOptions = { wallet: wallet, identity: userId, discovery: { enabled: true, asLocalhost: false } };
-		    let gateway = new Gateway();
-		    await gateway.connect(connectionProfile, connectionOptions);
-		    const network = await gateway.getNetwork(channelName);
-            return network;
-	    } catch (error) {
-            throw err;
-	    }
-    };
-    
-    // Logic to construct a security domain structure for this organization
-    // 'userId' represents a wallet identity
-    var getSecurityGroupMspConfig = async function(userId) {
-        const network = await getNetworkForOrgUser(userId);
-        logger.info('Getting MSP Info for org: ' + org + '.');
-        let mspConfig = network.getChannel().getMsp(mspId);
-        let securityGroupMspConfig = {};
-        securityGroupMspConfig.chain = [];
-        if (Array.isArray(mspConfig.rootCerts)) {
-            for (let i = 0; i < mspConfig.rootCerts.length ; i++) {
-                securityGroupMspConfig.chain.push(mspConfig.rootCerts[i]);
+- _Chaincode_: 
+  - _Data Sharing_: No code changes are required for Weaver enablement, as mentioned above.
+  - _Asset Exchange_: Asset exchange can be enabled in two ways, either use weaver `assetexchange` as library, or use interop chaincode to handle asset exchange by doing a chaincode to chaincode call.
```

Comment:
Wrap `assetexchange` with a link to the library package, here and in similar references below.